diff --git a/man-src/k_any/2-translated.Rmd b/man-src/k_any/2-translated.Rmd
--- a/man-src/k_any/2-translated.Rmd
+++ b/man-src/k_any/2-translated.Rmd
@@ -1,2 +1,2 @@
-Test whether any array element along a given axis evaluates to `True`.
+Test whether any array element along a given axis evaluates to `TRUE`.
 
@@ -5,19 +5,15 @@ Test whether any array element along a given axis evaluates to `True`.
 # Examples
-```python
-x = keras.ops.convert_to_tensor([True, False])
-keras.ops.any(x)
-# array(True, shape=(), dtype=bool)
+```{r}
+x <- k_constant(c(TRUE, FALSE))
+k_any(x)
 ```
 
-```python
-x = keras.ops.convert_to_tensor([[True, False], [True, True]])
-keras.ops.any(x, axis=0)
-# array([ True  True], shape=(2,), dtype=bool)
+```{r}
+(x <- k_convert_to_tensor(array(c(TRUE, FALSE, TRUE, TRUE, TRUE, TRUE), dim = c(3, 2))))
+k_any(x, axis = 1)
 ```
 
-`keepdims=True` outputs a tensor with dimensions reduced to one.
-```python
-x = keras.ops.convert_to_tensor([[True, False], [True, True]])
-keras.ops.all(x, keepdims=True)
-# array([[False]], shape=(1, 1), dtype=bool)
+`keepdims = TRUE` outputs a tensor with dimensions reduced to one.
+```{r}
+k_any(x, keepdims = TRUE)
 ```
@@ -30,8 +26,8 @@ The tensor containing the logical OR reduction over the `axis`.
     which a logical OR reduction is performed. The default
-    (`axis=None`) is to perform a logical OR over all the dimensions
+    (`axis = NULL`) is to perform a logical OR over all the dimensions
     of the input array. `axis` may be negative, in which case it counts
     for the last to the first axis.
-@param keepdims If `True`, axes which are reduced are left in the result as
+@param keepdims If `TRUE`, axes which are reduced are left in the result as
     dimensions with size one. With this option, the result will
-    broadcast correctly against the input array. Defaults to`False`.
+    broadcast correctly against the input array. Defaults to `FALSE`.
 
