diff --git a/man-src/k_arccosh/2-translated.Rmd b/man-src/k_arccosh/2-translated.Rmd
--- a/man-src/k_arccosh/2-translated.Rmd
+++ b/man-src/k_arccosh/2-translated.Rmd
@@ -5,6 +5,5 @@ Inverse hyperbolic cosine, element-wise.
 # Examples
-```python
-x = keras.ops.convert_to_tensor([10, 100])
-keras.ops.arccosh(x)
-# array([2.993223, 5.298292], dtype=float32)
+```{r}
+x <- k_convert_to_tensor(c(10, 100))
+k_arccosh(x)
 ```
