diff --git a/man-src/k_arange/2-translated.Rmd b/man-src/k_arange/2-translated.Rmd
--- a/man-src/k_arange/2-translated.Rmd
+++ b/man-src/k_arange/2-translated.Rmd
@@ -13,20 +13,8 @@ Return evenly spaced values within a given interval.
 # Examples
-```python
-keras.ops.arange(3)
-# array([0, 1, 2], dtype=int32)
-```
-
-```python
-keras.ops.arange(3.0)
-# array([0., 1., 2.], dtype=float32)
-```
-
-```python
-keras.ops.arange(3, 7)
-# array([3, 4, 5, 6], dtype=int32)
-```
-
-```python
-keras.ops.arange(3, 7, 2)
-# array([3, 5], dtype=int32)
+```{r}
+keras::k_arange(3L)
+keras::k_arange(3) # float
+keras::k_arange(3, dtype = 'int32') #int
+keras::k_arange(3L, 7L)
+keras::k_arange(3L, 7L, 2L)
 ```
@@ -36,3 +24,3 @@ Tensor of evenly spaced values.
 For floating point arguments, the length of the result is
-`ceil((stop - start)/step)`. Because of floating point overflow, this
+`ceiling((stop - start)/step)`. Because of floating point overflow, this
 rule may result in the last element of out being greater than stop.
@@ -44,3 +32,3 @@ rule may result in the last element of out being greater than stop.
     `step` is not an integer and floating point round-off affects the
-    lenght of `out`. Defaults to `None`.
+    length of `out`. Defaults to `NULL`.
 @param step Integer or real, represent the spacing between values. For any
