diff --git a/man-src/k_append/2-translated.Rmd b/man-src/k_append/2-translated.Rmd
--- a/man-src/k_append/2-translated.Rmd
+++ b/man-src/k_append/2-translated.Rmd
@@ -5,7 +5,6 @@ Append tensor `x2` to the end of tensor `x1`.
 # Examples
-```python
-x1 = keras.ops.convert_to_tensor([1, 2, 3])
-x2 = keras.ops.convert_to_tensor([[4, 5, 6], [7, 8, 9]])
-keras.ops.append(x1, x2)
-# array([1, 2, 3, 4, 5, 6, 7, 8, 9], dtype=int32)
+```{r}
+x1 <- k_convert_to_tensor(c(1, 2, 3))
+x2 <- k_convert_to_tensor(rbind(c(4, 5, 6), c(7, 8, 9)))
+k_append(x1, x2)
 ```
@@ -13,15 +12,8 @@ keras.ops.append(x1, x2)
 When `axis` is specified, `x1` and `x2` must have compatible shapes.
-```python
-x1 = keras.ops.convert_to_tensor([[1, 2, 3], [4, 5, 6]])
-x2 = keras.ops.convert_to_tensor([[7, 8, 9]])
-keras.ops.append(x1, x2, axis=0)
-# array([[1, 2, 3],
-#         [4, 5, 6],
-#         [7, 8, 9]], dtype=int32)
-x3 = keras.ops.convert_to_tensor([7, 8, 9])
-keras.ops.append(x1, x3, axis=0)
-# Traceback (most recent call last):
-#     ...
-# TypeError: Cannot concatenate arrays with different numbers of
-# dimensions: got (2, 3), (3,).
+```{r}
+x1 <- k_convert_to_tensor(rbind(c(1, 2, 3), c(4, 5, 6)))
+x2 <- k_convert_to_tensor(rbind(c(7, 8, 9)))
+k_append(x1, x2, axis = 1)
+x3 <- k_convert_to_tensor(c(7, 8, 9))
+try(k_append(x1, x3, axis = 1))
 ```
@@ -34,3 +26,3 @@ A tensor with the values of `x2` appended to `x1`.
 @param axis Axis along which tensor `x2` is appended to tensor `x1`.
-    If `None`, both tensors are flattened before use.
+    If `NULL`, both tensors are flattened before use.
 
