diff --git a/man-src/k_arccos/2-translated.Rmd b/man-src/k_arccos/2-translated.Rmd
--- a/man-src/k_arccos/2-translated.Rmd
+++ b/man-src/k_arccos/2-translated.Rmd
@@ -6,6 +6,5 @@ The inverse of `cos` so that, if `y = cos(x)`, then `x = arccos(y)`.
 # Examples
-```python
-x = keras.ops.convert_to_tensor([1, -1])
-keras.ops.arccos(x)
-# array([0.0, 3.1415927], dtype=float32)
+```{r}
+x <- k_onvert_to_tensor(c(1, -1))
+k_arccos(x)
 ```
