diff --git a/man-roxygen/activation_elu/roxygen.Rmd b/man-roxygen/activation_elu/roxygen.Rmd
index 5f24ad70..1bf8111a 100644
--- a/man-roxygen/activation_elu/docstring_as_roxygen.md
+++ b/man-roxygen/activation_elu/roxygen.Rmd
@@ -3,6 +3,6 @@ Exponential Linear Unit.
 @description
-The exponential linear unit (ELU) with `alpha > 0` is define as:
+The exponential linear unit (ELU) with `alpha > 0` is defined as:
 
 - `x` if `x > 0`
-- alpha * `exp(x) - 1` if `x < 0`
+- `alpha * exp(x) - 1` if `x < 0`
 
