diff --git a/man-src/initializer_identity/2-translated.Rmd b/man-src/initializer_identity/2-translated.Rmd
--- a/man-src/initializer_identity/2-translated.Rmd
+++ b/man-src/initializer_identity/2-translated.Rmd
@@ -6,12 +6,12 @@ Only usable for generating 2D matrices.
 # Examples
-```python
+```{r}
 # Standalone usage:
-initializer = Identity()
-values = initializer(shape=(2, 2))
+initializer <- initializer_identity()
+values <- initializer(shape = c(2, 2))
 ```
 
-```python
+```{r}
 # Usage in a Keras layer:
-initializer = Identity()
-layer = Dense(3, kernel_initializer=initializer)
+initializer <- initializer_identity()
+layer <- layer_dense(units = 3, kernel_initializer = initializer)
 ```
