diff --git a/man-src/k_arcsin/2-translated.Rmd b/man-src/k_arcsin/2-translated.Rmd
--- a/man-src/k_arcsin/2-translated.Rmd
+++ b/man-src/k_arcsin/2-translated.Rmd
@@ -5,6 +5,5 @@ Inverse sine, element-wise.
 # Examples
-```python
-x = keras.ops.convert_to_tensor([1, -1, 0])
-keras.ops.arcsin(x)
-# array([ 1.5707964, -1.5707964,  0.], dtype=float32)
+```{r}
+x <- k_convert_to_tensor(c(1, -1, 0))
+k_arcsin(x)
 ```
