diff --git a/man-src/config_set_floatx/2-translated.Rmd b/man-src/config_set_floatx/2-translated.Rmd
index edbff4cf..a43703fb 100644
--- a/man-src/config_set_floatx/2-translated.Rmd
+++ b/man-src/config_set_floatx/2-translated.Rmd
@@ -7,19 +7,17 @@ Instead, mixed precision, which leverages
 a mix of `float16` and `float32`. It can be configured by calling
-`keras.mixed_precision.set_dtype_policy('mixed_float16')`.
+`keras::keras$mixed_precision$set_dtype_policy('mixed_float16')`.
 
 # Examples
-```python
-keras.config.floatx()
-# 'float32'
+```{r}
+config_floatx()
 ```
 
-```python
-keras.config.set_floatx('float64')
-keras.config.floatx()
-# 'float64'
+```{r}
+config_set_floatx('float64')
+config_floatx
 ```
 
-```python
+```{r}
 # Set it back to float32
-keras.config.set_floatx('float32')
+config_set_floatx('float32')
 ```
