diff --git a/man-src/config_image_data_format/2-translated.Rmd b/man-src/config_image_data_format/2-translated.Rmd
--- a/man-src/config_image_data_format/2-translated.Rmd
+++ b/man-src/config_image_data_format/2-translated.Rmd
@@ -5,5 +5,4 @@ Return the default image data format convention.
 # Examples
-```python
-keras.config.image_data_format()
-# 'channels_last'
+```
+keras::config_image_data_format()
 ```
