diff --git a/man-src/initializer_zeros/2-translated.Rmd b/man-src/initializer_zeros/2-translated.Rmd
--- a/man-src/initializer_zeros/2-translated.Rmd
+++ b/man-src/initializer_zeros/2-translated.Rmd
@@ -5,12 +5,12 @@ Initializer that generates tensors initialized to 0.
 # Examples
-```python
+```{r}
 # Standalone usage:
-initializer = Zeros()
-values = initializer(shape=(2, 2))
+initializer <- initializer_zeros()
+values <- initializer(shape = c(2, 2))
 ```
 
-```python
+```{r}
 # Usage in a Keras layer:
-initializer = Zeros()
-layer = Dense(units=3, kernel_initializer=initializer)
+initializer <- initializer_zeros()
+layer <- layer_dense(units = 3, kernel_initializer = initializer)
 ```
