diff --git a/man-src/initializer_constant/2-translated.Rmd b/man-src/initializer_constant/2-translated.Rmd
--- a/man-src/initializer_constant/2-translated.Rmd
+++ b/man-src/initializer_constant/2-translated.Rmd
@@ -8,15 +8,15 @@ when calling the initializer.
 # Examples
-```python
+```{r}
 # Standalone usage:
-initializer = Constant(10.)
-values = initializer(shape=(2, 2))
+initializer <- initializer_constant(10)
+values <- initializer(shape = c(2, 2))
 ```
 
-```python
+```{r}
 # Usage in a Keras layer:
-initializer = Constant(10.)
-layer = Dense(3, kernel_initializer=initializer)
+initializer <- initializer_constant(10)
+layer <- layer_dense(units = 3, kernel_initializer = initializer)
 ```
 
-@param value A Python scalar.
+@param value A numeric scalar.
 
