diff --git a/man-src/get_file/2-translated.Rmd b/man-src/get_file/2-translated.Rmd
index fbe86514..4cc051a1 100644
--- a/man-src/get_file/2-translated.Rmd
+++ b/man-src/get_file/2-translated.Rmd
@@ -14,6 +14,6 @@ programs `shasum` and `sha256sum` can compute the hash.
 # Examples
-```python
-path_to_downloaded_file = get_file(
-    origin="https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz",
-    extract=True,
+```{r}
+path_to_downloaded_file <- get_file(
+    origin = "https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz",
+    extract = TRUE
 )
@@ -34,3 +34,3 @@ are getting is the one you expect.
     is specified, the file will be saved at that location.
-    If `None`, the name of the file at `origin` will be used.
+    If `NULL`, the name of the file at `origin` will be used.
 @param origin Original URL of the file.
@@ -48,11 +48,10 @@ are getting is the one you expect.
     The default 'auto' detects the hash algorithm in use.
-@param extract True tries extracting the file as an Archive, like tar or zip.
+@param extract `TRUE` tries extracting the file as an Archive, like tar or zip.
 @param archive_format Archive format to try for extracting the file.
-    Options are `"auto'`, `"tar'`, `"zip'`, and `None`.
+    Options are `"auto'`, `"tar'`, `"zip'`, and `NULL`.
     `"tar"` includes tar, tar.gz, and tar.bz files.
-    The default `"auto"` corresponds to `["tar", "zip"]`.
-    None or an empty list will return no matches found.
-@param cache_dir Location to store cached files, when None it
-    defaults ether `$KERAS_HOME` if the `KERAS_HOME` environment
-    variable is set or `~/.keras/`.
+    The default `"auto"` corresponds to `c("tar", "zip")`.
+    `NULL` or an empty list will return no matches found.
+@param cache_dir Location to store cached files, when `NULL` it
+    defaults to `Sys.getenv("KERAS_HOME", '~/.keras/')`.
 
