diff --git a/man-src/callback_csv_logger/2-translated.Rmd b/man-src/callback_csv_logger/2-translated.Rmd
--- a/man-src/callback_csv_logger/2-translated.Rmd
+++ b/man-src/callback_csv_logger/2-translated.Rmd
@@ -4,8 +4,8 @@ Callback that streams epoch results to a CSV file.
 Supports all values that can be represented as a string,
-including 1D iterables such as `np.ndarray`.
+including 1D iterables such as atomic vectors.
 
 # Examples
-```python
-csv_logger = CSVLogger('training.log')
-model.fit(X_train, Y_train, callbacks=[csv_logger])
+```r
+csv_logger <- callback_csv_logger('training.log')
+model %>% fit(X_train, Y_train, callbacks = list(csv_logger))
 ```
@@ -14,4 +14,4 @@ model.fit(X_train, Y_train, callbacks=[csv_logger])
 @param separator String used to separate elements in the CSV file.
-@param append Boolean. True: append if file exists (useful for continuing
-    training). False: overwrite existing file.
+@param append Boolean. `TRUE`: append if file exists (useful for continuing
+    training). `FALSE`: overwrite existing file.
 
