diff --git a/man-src/image_load/2-translated.Rmd b/man-src/image_load/2-translated.Rmd
--- a/man-src/image_load/2-translated.Rmd
+++ b/man-src/image_load/2-translated.Rmd
@@ -5,7 +5,14 @@ Loads an image into PIL format.
 # Usage
-```python
-image = keras.utils.load_img(image_path)
-input_arr = keras.utils.img_to_array(image)
-input_arr = np.array([input_arr])  # Convert single image to a batch.
-predictions = model.predict(input_arr)
+```{r}
+image_path <- get_file(origin = "https://www.r-project.org/logo/Rlogo.png")
+(image <- image_load(image_path))
+
+input_arr <- image_to_array(image)
+str(input_arr)
+input_arr %<>% array_reshape(dim = c(1, dim(input_arr))) # Convert single image to a batch.
+```
+
+
+```{r, eval = FALSE}
+model |> predict(input_arr)
 ```
@@ -18,3 +25,3 @@ predictions = model.predict(input_arr)
     The desired image format.
-@param target_size Either `None` (default to original size) or tuple of ints
+@param target_size Either `NULL` (default to original size) or tuple of ints
     `(img_height, img_width)`.
